bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -106,7 +107,10 @@ final class BagsManager(executorService: ExecutorService) {
         case AutomaticGroup.Initial | AutomaticGroup.Active(_,_) => true
         case _                                                   => false
       }
-      enabledGroup && (ctx.getInstrument.getType != SPComponentType.INSTRUMENT_GPI)
+      enabledGroup &&
+        (ctx.getInstrument.getType != SPComponentType.INSTRUMENT_GPI) &&
```

Comment:
I think that that's a good idea, but I'm going to create a separate JIRA task for it, since it is outside of the scope of this task.